addons/pinniped/config-controller/controllers/handlers.go

```diff
@@ -14,12 +14,14 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/event"
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
-
-	"github.com/vmware-tanzu/tanzu-framework/addons/pinniped/config-controller/constants"
-	"github.com/vmware-tanzu/tanzu-framework/addons/pinniped/config-controller/utils"
 )
 
-func (c *PinnipedController) configMapToSecret(o client.Object) []ctrl.Request {
+func (c *PinnipedV3Controller) configMapToSecret(o client.Object) []ctrl.Request {
```

Comment:
does `configMapToSecret()` need to be defined on a struct? could we just use the same function for both controllers?

Reply:
mmm good call, it really doesn't.  renaming it and using it for both!